force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -239,10 +293,11 @@ public with sharing class FlowControlLoopsCollections {
     public static Set<String> createStringSet(List<String> inputList) {
         // Create a new Set of strings
         Set<String> stringSet = new Set<String>();
-
+        for(String str : inputList){
         //add strings to the set
-
-        return null; // Replace null with the variable you used to store the result
+            stringSet.add(str);
+        }
+        return stringSet; // Replace null with the variable you used to store the result
```

Comment:
You can create a Set from a List. It has a special constructor: 
`Set<String> myUniqueValues = new Set<String>(inputList);`